samples/benchmarks/coremark/Kconfig.sysbuild

```
@@ -0,0 +1,16 @@
+#
+# Copyright (c) 2024 Nordic Semiconductor ASA
+#
+# SPDX-License-Identifier: LicenseRef-Nordic-5-Clause
+#
+
+source "share/sysbuild/Kconfig"
+rsource "boards/Kconfig.sysbuild_boards"
+
+config IMAGE_2_BOARD
+	string
+	default ""
+
+config IMAGE_3_BOARD
+	string
+	default ""
```

Can we have something like this?

```suggestion
if SOC_SERIES_NRF53X
rsource "boards/Kconfig.sysbuild_nrf53_series"
endif

config IMAGE_2_BOARD
	string
	default ""

config IMAGE_3_BOARD
	string
	default ""
```

---

The case is, we still do not support all of the boards with given SoC (we must rely on board names anyway). Because of that we can even keep it as is and get back to it once we

---

I like the idea of keeping all boards in one file.